jobs/backfill-ghost-explore-follows/index.ts

```
@@ -0,0 +1,368 @@
+        // Create headers
+        const headers: Record<string, string> = {
+            'Host': targetAccount.ap_inbox_url.host,
+            'Date': new Date().toUTCString(),
+            'Content-Type': 'application/activity+json',
+            'Accept': 'application/activity+json',
+        };
+
+        // Add Digest header (SHA-256 of the body)
+        const bodyBuffer = Buffer.from(body);
+        const digest = createHash('sha256').update(bodyBuffer).digest('base64');
+        headers['Digest'] = `SHA-256=${digest}`;
```

Biome (1.9.4)

[error] The computed expression can be simplified without the use of a string literal.

Unsafe fix: Use a literal key instead.


(lint/complexity/useLiteralKeys)